moveit_core/constraint_samplers/src/default_constraint_samplers.cpp

```diff
@@ -91,14 +92,14 @@ bool JointConstraintSampler::configure(const std::vector<kinematic_constraints::
         std::max(joint_bounds.min_position_, jc[i].getDesiredJointPosition() - jc[i].getJointToleranceBelow()),
         std::min(joint_bounds.max_position_, jc[i].getDesiredJointPosition() + jc[i].getJointToleranceAbove()));
 
-    ROS_DEBUG_NAMED("constraint_samplers", "Bounds for %s JointConstraint are %g %g",
+    RCLCPP_DEBUG(LOGGER_DEFAULT_CONTRAINT_SAMPLERS, "Bounds for %s JointConstraint are %g %g",
                     jc[i].getJointVariableName().c_str(), ji.min_bound_, ji.max_bound_);
 
     if (ji.min_bound_ > ji.max_bound_ + std::numeric_limits<double>::epsilon())
     {
       std::stringstream cs;
       jc[i].print(cs);
-      ROS_ERROR_NAMED("constraint_samplers",
+      RCLCPP_ERROR(LOGGER_DEFAULT_CONTRAINT_SAMPLERS,
```

Comment:
Be sure to run clang every time

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/abf4a74859bb5f3ca50a946b9bd0f48f1db59fa5